temporalio/worker/workflow_instance.py

```diff
@@ -589,6 +600,11 @@ async def run_workflow(input: ExecuteWorkflowInput) -> None:
             self._run_top_level_workflow_function(run_workflow(input))
         )
 
+    def _apply_update_random_seed(
+        self, job: temporalio.bridge.proto.workflow_activation.UpdateRandomSeed
+    ) -> None:
+        self._random = random.Random(job.randomness_seed)
```

Comment:
I'd probably reseed the current `_random` reference in case users keep a reference to it
(https://docs.python.org/3/library/random.html#random.seed)

Reply:
Ah, of course, duh. Fixing.